features/0025-didcomms-transports/README.md

```diff
@@ -0,0 +1,81 @@
+
+## Reference
+
+Standardized transport methods are detailed here. 
+
+### HTTP(S)
+
+HTTP(S) is the first transport for DID Communication that has received heavy attention.
+
+- Messages are transported via HTTP POST.
+- The MIME Type for the POST request is `application/ssi-agent-wire`.
+- A received message should be responded to with a 202 Accepted status code. This indicates that the request was received, but not necessarily processed. Accepting a 200 OK status code is allowed.
+- POST requests are considered transmit only by default. No agent messages will be returned in the response. This behavior may be modified with additional signaling.
+- Using HTTPS with TLS 1.2 or greater with a forward secret cipher will provide Perfect Forward Secrecy (PFS) on the transmission leg.
+
+#### Known Implementations
+
+[Python Reference Agent](<https://github.com/hyperledger/indy-agent/tree/master/python>)
+
+[Indy Catalyst Agent](<https://github.com/bcgov/indy-catalyst/tree/master/agent>)
+
+### Websocket
+
+Websockets are an efficient way to transmit multiple messages without the overhead of individual requests. 
+
+- Each message is transmitted individually in an Encryption Envelope.
+- The trust of each message comes from the Encryption Envelope, not the socket connection itself.
+- Websockets are considered transmit only by default. Messages will only flow from the agent that opened the socket. This behavior may be modified with additional signaling.
+- Using Secure Websockets (wss://) with TLS 1.2 or greater with a forward secret cipher will provide Perfect Forward Secrecy (PFS) on the transmission leg.
+
+#### Known Implementations
+
+[Python Reference Agent](<https://github.com/hyperledger/indy-agent/tree/master/python>)
+
+
+### Other Transports
+
+Other transports may be used for Agent messaging. As they are developed, this HIPE should be updated with appropriate standards for the transport method. A PR should be raised against this doc to facilitate discussion of the proposed additions and/or updates. New transports should highlight the common elements of the transport (such as an HTTP response code for the HTTP transport) and how they should be applied.
+
+### Message Routing
+
+The transports described here are used between two agents. In the case of [message routing](<https://github.com/hyperledger/indy-hipe/tree/master/text/0022-cross-domain-messaging>), a message will travel across multiple agent connections. Each intermediate agent (see [Mediators and Relays](<https://github.com/hyperledger/indy-hipe/tree/master/text/0036-mediators-and-relays>)) may use a different transport. These transport details are not made known to the sender, who only knows the keys of Mediators and the first endpoint of the route. 
```

Comment:
Are these referenced HIPES being moved to RFCs?

Reply:
I will be moving the one about mediators and relays.